mapper/mapper.go

```diff
@@ -22,3 +22,39 @@ func CreateStartPage(basePage coreModel.Page) StartPageModel {
 	model.Page.BetaBannerEnabled = true
 	return model
 }
+
+type AreaModel struct {
+	coreModel.Page
+	Name string `json:"name"`
+	Level string `json:"level"`
+	Code string `json:"code"`
+	Ancestors []AreaModel `json:"ancestors"`
+	Siblings []AreaModel `json:"siblings"`
+	Children []AreaModel `json:"children"`
+}
+
+func CreateAreaPage(basePage coreModel.Page) AreaModel {
+	// TODO - load the area data for the requested area once the API has been developed
+	model := AreaModel{
+		Page: basePage,
+	}
+	model.Metadata = coreModel.Metadata{
+		Title: model.Name + " Summary",
```

Comment:
Using `fmt.Sprintf` would be cleaner here
```suggestion
		Title: fmt.Sprintf("%s Summary", model.Name),
```

Reply:
This is a lot cleaner